release/nightly_tests/shuffle/shuffle_app_config.yaml

```diff
@@ -0,0 +1,9 @@
+base_image: "anyscale/ray-ml:pinned-nightly"
+env_vars: {}
+debian_packages: []
+
+python:
+  pip_packages: []
```

Comment:
I think we need tqdm to make it work? (or is it already included in the image?)

Reply:
I think everything's there in ray-ml by default